src/app/sentry/capture.ts

```diff
@@ -0,0 +1,5 @@
+import * as Sentry from "@sentry/browser"
+
+export default (message: string) => {
+  Sentry.captureException(new Error(message))
```

Comment:
Where/how is this used?

Reply:
Just a convenience method

Reply:
Is it ever called?

Reply:
Not yet. Initializing Sentry will already pick up and logs all runtime errors. You could add more custom logging using this function.

Reply:
But will remove